app/models/sales_log.rb

```diff
@@ -223,7 +224,7 @@ def postcode_full=(postcode)
   def expected_shared_ownership_deposit_value
     return unless value && equity
 
-    (value * equity / 100).round(2)
+    sprintf("%.2f", (value * equity / 100))
```

Comment:
wouldn't this also just like to use something from the MoneyForattingHelper rather than sprintf?